Airline_Web_API/Airline_Web_API/Models/TicketType.cs

```diff
@@ -9,13 +9,13 @@ public class TicketType
     {
         public int Id { get; set; }
         public string Name { get; set; }
-        public int? CarryOnBagNumber { get; set; }
+        public int? CarryOnBagsNumber { get; set; }
         public int? CarryOnBagMaxWeight { get; set; }
         public int? BaggageNumber { get; set; }
         public int? BaggageMaxWeight { get; set; }
         public bool SeatReservation { get; set; }
         public bool Changes { get; set; }
-        public string Refund { get; set; }
+        public bool? Refund { get; set; }
```

Comment:
Why would you need null value here? What is the meaning of it?

Reply:
Null - if there is no refund for this type of ticket, false - refund for fee, true - free refund

Reply:
I'm approving it for now, but please consider using more appropriate data type here instead.